src/classes/STG_SettingsManager_TEST.cls

```diff
@@ -240,39 +240,266 @@ public with sharing class STG_SettingsManager_TEST {
     * @description Tests the Households panel
     * @return void
     */
-    public static testMethod void testPanelHouseholds() {
-        if (strTestOnly != '*' && strTestOnly != 'testPanelHouseholds') return;
+    public static testMethod void STG_PanelHouseholds_testPanelHouseholds() {
+        if (strTestOnly != '*' && strTestOnly != 'STG_PanelHouseholds_testPanelHouseholds') return;
+
         STG_PanelHouseholds_CTRL panel = new STG_PanelHouseholds_CTRL();
         system.assertEquals('idPanelHH', panel.idPanel());
+
         STG_Panel.stgService.stgHH.npo02__Advanced_Household_Naming__c = false;
         panel.saveSettings();
         system.assertEquals(false, UTIL_CustomSettingsFacade.getOrgHouseholdsSettings().npo02__Advanced_Household_Naming__c);
     }
 
     /*********************************************************************************************************
-    * @description Tests that the name specs are valid in the Households panel
-    * @return void
-    */
-    public static testMethod void testNameSpecs() {
-        if (strTestOnly != '*' && strTestOnly != 'testNameSpecs') return;
-        STG_PanelHouseholds_CTRL panel = new STG_PanelHouseholds_CTRL();
+    @description 
```

Comment:
given the amount of test you've added just for the household panel, I think it would be better to have them in their own class file.  this would also alleviate the need to prefix each test with STG_Panelhouseholds_